device/transport/amqp/src/amqp.ts

```diff
@@ -259,11 +218,12 @@ export class Amqp extends EventEmitter implements Client.Transport, StableConnec
             callback();
           },
           onDeviceMethod: (methodName, callback) => {
+            // TODO - confirm this is still what we want now that we have enableMethods
```

Comment:
>// TODO - confirm this is still what we want now that we have enableMethods [](start = 12, length = 75)

!

Reply:
yup, should've removed that as well as the call the enableMethods below. very very good catch.

---
In reply to: [145259295](https://github.com/Azure/azure-iot-sdk-node/pull/109#discussion_r145259295) [](ancestors = 145259295)